qpc/messages.py

```diff
@@ -308,6 +308,7 @@
 INSIGHTS_TMP_ERROR = 'An error occurred while saving a temporary copy of' \
     ' the QPC report.  Cannot write file to %s.'
 INSIGHTS_REQUIRE_SUDO = 'Insights upload command requires sudo access.'
+INSIGHTS_NO_GPG_HELP = 'GNU Privacy Gaurd is not installed.'
```

Comment:
Change `Gaurd` to `Guard`

Reply:
On second thought, this message should be something more along the lines of: 
```Upload to insights with GNU Privacy Guard not installed``` or ```Upload to insights without GNU Privacy Guard```